src/services/kuzzleWrapper-v1.ts

```diff
@@ -14,13 +20,21 @@ let getValueAdditionalAttribute = (content, attributePath) => {
 
   return content[attribute]
 }
-const formatMeta = _kuzzle_info => ({
-  author: _kuzzle_info.author === '-1' ? 'Anonymous (-1)' : _kuzzle_info.author,
-  updater:
-    _kuzzle_info.updater === '-1' ? 'Anonymous (-1)' : _kuzzle_info.updater,
-  createdAt: _kuzzle_info.createdAt,
-  updatedAt: _kuzzle_info.updatedAt
-})
+
+function buildCaseInsensitiveRegexp(searchString) {
+  return searchString
+    .split('')
+    .map(e => {
```

Comment:
Nitpicking: What does this `e` stand for? Is it for an error, an event or an element?